cmd/ci-images-mirror/main.go

```diff
@@ -183,6 +192,16 @@ func main() {
 		logrus.WithError(err).Fatal("Failed to validate options")
 	}
 
+	if opts.validateConfigOnly {
+		ciOperatorConfigPath := filepath.Join(opts.releaseRepoGitSyncPath, config.CiopConfigInRepoPath)
```

Comment:
NIT:
We could defined it only once for 
https://github.com/openshift/ci-tools/blob/fbee3dfe3b0757b98cd2d6f49838621e3e69f757/cmd/ci-images-mirror/main.go#L279 as well.